mappings/net/minecraft/entity/mob/EntityCaveSpider.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS abk net/minecraft/entity/mob/EntityCaveSpider
 	METHOD J getLootTableId ()Lmd;
 	METHOD a registerTagUpgraders (Lqv;)V
+	METHOD a (Lsv;Lun;)Lun;
```

Comment:
This was one of the parameter names I had difficulty reverting, I must have missed this one. If it's important to remove, I'll fix up this PR when I get a chance later.